crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -5393,7 +5393,7 @@ mod tests {
         assert_file_diagnostics(
             &db,
             "/src/a.py",
-            &[r"Conflicting declared types for 'x': 'str', 'int'."],
+            &[r"Conflicting declared types for 'x': str, int."],
```

Comment:
Is it intentional that the types are no longer quoted?

Reply:
Yeah, the more I looked at it the more it just seemed like visual noise.